tools/src/tokenize.rs

```diff
@@ -291,7 +291,7 @@ pub fn tokenize_c_like(string: &str, spec: &LanguageSpec) -> Vec<Token> {
             tokens.push(Token {start: start, end: nl, kind: TokenKind::Comment});
             tokens.push(Token {start: nl, end: peek_pos(), kind: TokenKind::Newline});
         } else if ch == '#' && spec.c_preprocessor {
-            while peek_char() == ' ' {
+            while peek_char() == ' ' || peek_char() == '\t' {
```

Comment:
Should these also handle other kinds of whitespace?

Reply:
Good question.
According to this line: https://github.com/mozsearch/mozsearch/blob/master/tools/src/tokenize.rs#L27
whitespace could be a "\r" but could we really have such a char after a "#" ?

Reply:
Probably. Anyhow this can probably be a follow-up. Let's get this merged.